axelor-stock/src/main/java/com/axelor/apps/stock/web/StockLocationController.java

```diff
@@ -141,10 +141,11 @@ public void print(ActionRequest request, ActionResponse response) throws AxelorE
   public void setStocklocationValue(ActionRequest request, ActionResponse response) {
 
     StockLocation stockLocation = request.getContext().asType(StockLocation.class);
-
-    response.setValue(
-        "stockLocationValue",
-        Beans.get(StockLocationService.class).getStockLocationValue(stockLocation));
+    if (stockLocation.getTypeSelect() != StockLocationRepository.TYPE_VIRTUAL) {
+      response.setValue(
+          "stockLocationValue",
+          Beans.get(StockLocationService.class).getStockLocationValue(stockLocation));
+    }
   }
```

Comment:
if condition is added to solve issue of star(*) on stock-location form view having type = virtual